### ISSUE TITLE
Urgent : Major bugs introduced in 0.15

### ISSUE THREAD

**Issue by rwatts3** [tststs/atom-ternjs#248]
Greetings, 

There were some major performance issues following the release of `0.15.0` . 
I've noticed a significant increase in package load time. It used to be `70ms` and now it's `1618ms`. 
Also with the package activated, the second I attempt to start typing within a comment block atom crashes ,

Two redundant entries in the config settings for Display Above Snippets and one for Display above Snippets.
Both appear to be doing two completely different things.

Great work on the project, but i'm afraid the release of 0.15 has really destroyed my productivity for the day. 

http://screencast.com/t/43cXXxFkQOj

---
OS : Windows 7
Atom : 1.10 beta7

**Comment by tststs** [comment]
Downgrade to 0.14.2 and i'll have a look into this later.

- Does the performance issue only affects startup time?
- The "Display Above Snippets" is atom related. The only workaround is to delete tern related config data and reinstall the package

There is no duplicate entry here:

![bildschirmfoto 2016-08-26 um 09 19 19](https://cloud.githubusercontent.com/assets/3591665/17997373/6d1d71d8-6b6e-11e6-84fb-1b1697b11e73.png)

**Comment by tinchoz49** [comment]
Hi @tststs, how are you? 
I'm having the same issue. I think is only affects the startup, the autocomplete is working excellent in my case. Maybe is a sync operation that is taking some time on the startup (read a config file, check something, ...)

![screenshot_2016-08-26_09-47-30](https://cloud.githubusercontent.com/assets/819446/18005915/4e077cbc-6b73-11e6-9d2b-43dafd03d35b.png)

**Comment by tststs** [comment]
yes, startup time has increased and i will try to to reduce it in the future, but this is a low level priority for now. i switched mostly to import syntax and the package imports all dependencies on startup. This makes life a bit easier.

@rwatts3 you have example code that crashes the editor? i wasn't able to reproduce the issue in any of my projects.

greetings

**Comment by mhnavarro** [comment]
Hi

This package was working perfectly before this update. Now lint is not working anymore,
this notification pops up when i restart the server
![image](https://cloud.githubusercontent.com/assets/7016419/18039501/e56d8404-6d60-11e6-9a82-2ae5e9ccdecd.png)

**Comment by eyesofkids** [comment]
the new version(v0.15) has removed the lint plugin.
I have the same problem today and resolved it. 
You can delete the old .tern-project file and re-generate new one or just delete this lint plugin config in your .tern-project file.

```
"lint": {},
```

**Comment by tststs** [comment]
`tern-lint` isn't supported at the moment and was removed in favor for tern@0.20.0.
this is due the fact that `tern-lint` wasn't updated in a while and does not seem to be compatible with the current version of Tern `atom-ternjs` uses.

**Comment by TheAncientGoat** [comment]
It seems that tern-lint has tern 0.20.0 support now? https://github.com/angelozerr/tern-lint/issues/73

**Comment by jiayihu** [comment]
I'm having the same issue and currently `atom-ternjs` is the slowest to activate with about ~300ms. Any improvement on the performance would be much appreciated since I love using ternjs. 
Anyway thanks for your amazing work so far (:
